userv2/user_contact.go

```diff
@@ -0,0 +1,9 @@
+package userv2
+
+// UserContact contains data of user contact.
+type UserContact struct {
+	ID            string `json:"id"`
+	To            string `json:"to"`
+	ContactMethod string `json:"contactMethod"`
```

Comment:
Awesome. I was more meaning to use the Constants as the Type of the `ContactMethod` field in the Struct, in place of String?